scripts/tuple_tree_generator/tuple_tree_generator/generators/python.py

```diff
@@ -158,6 +165,45 @@ def to_bool(self, input: bool):
             return 'True'
         return 'False'
 
+    def add_mixins(self, formatted_rendered_template: str):
+        # Custom unpraser that doesn't remove backslashes
+        class Unparser(ast._Unparser):
```

Comment:
By default calling `ast.unprase` doesn't parse `\n` as new lines so instead of this:
```
class ABI(EnumBase):
    Invalid = 'Invalid'
    """64-bit SystemV ABI for x86 processor architecture.
    The latest version of the documentation can be found
    \\sa https://gitlab.com/x86-psABIs/x86-64-ABI
    """
```
You get this:
```
class ABI(EnumBase):
    Invalid = 'Invalid'
    '64-bit SystemV ABI for x86 processor architecture.\n    The latest version of the documentation can be found\n    \\sa https://gitlab.com/x86-psABIs/x86-64-ABI\n    '
```
I don't know how much we care about the formatting in the `_generated.py`, if we don't then we can go without a custom `unparser`

Reply:
As said in the comment below, the `ast` module should be run on `mixins.py` exclusively and the inheritance put in the template, not on the rendered template

Reply:
Actually this is not relevant anymore because I add `mixins` with filter so the formatting remains intact.